scripts/pipeline/static-linter-scan.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+readonly usage="Usage: static-linter-scan.sh --git-token <GHE token> --static-linter-version <Linter Version>"
+readonly WORK_DIR="${PWD}/linter-tool"
+
+function install_linter() {
+
+    # Download JSON of all versions
+    curl -H "Authorization: token ${GIT_TOKEN}" -H "Accept: application/vnd.github.v3.raw" -s https://github.ibm.com/api/v3/repos/IBMPrivateCloud/content-verification/releases > "${WORK_DIR}/versions.json"
+    
+    # Using jq, parse list of versions to find download for the version required
+    # Linter version is hard-coded here, for now
+    LINTER_TOOL_URL=$(jq -r '.[] | .assets | .[] |  select (.browser_download_url | endswith("v3.0.1/cv-linux-amd64.tar.gz")) | .url' "${WORK_DIR}/versions.json")
+    
+    # Use wget to download the linter binary, confirm it was successful
+    wget --user token --password $GIT_TOKEN --auth-no-challenge --header "Accept: application/octet-stream" $LINTER_TOOL_URL -O $WORK_DIR/cv.tar.gz
+    rc=$?
+    if [ $rc -ne 0 ]; then
+        # echo "Linter version ${STATIC_LINTER_VERSION} could not be located"
+        echo "Linter version 3.0.1 could not be located"
```

Comment:
Along with the above change to fix jq
```suggestion
        echo "Linter version ${STATIC_LINTER_VERSION} could not be located"
```

Reply:
Changes have been applied